### DISCUSSION TITLE
Could closing the 3DS lid send a button combination for rom to interpret?

### DISCUSSION THREAD

**Discussion by WizardWell** [profi200/open_agb_firm#149] [Ideas]
I found this topic regarding AGB_FIRM: https://github.com/LumaTeam/Luma3DS/issues/1691

Would it be possible to send a button combination when the lid closes and opens? Then you just have to add that to the GBA roms and you simulate sleep mode perfectly.

**Comment by profi200** [comment]
It's not quite that easy. I explicitly said in the README that sleep mode is not yet implemented. So while i can trigger a button combo on lid close this will only make the game halt but not make the system go to sleep mode yet.
